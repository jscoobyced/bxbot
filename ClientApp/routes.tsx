import * as React from 'react';
import { Route } from 'react-router-dom';
import { Layout } from './components/Layout';
import { Home } from './components/Home';
import { CandleChart } from './components/CandleChart';
import { Counter } from './components/Counter';

export const routes = <Layout>
    <Route exact path='/' component={ Home } />
    <Route path='/counter' component={ Counter } />
    <Route path='/candle' component={ CandleChart } />
</Layout>;
