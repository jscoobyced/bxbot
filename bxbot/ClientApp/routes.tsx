import * as React from 'react';
import { Route } from 'react-router-dom';
import { Layout } from './components/Layout';
import { Home } from './components/Home';
import { CandleChartPage } from './components/charts/CandleChartPage';

export const routes = <Layout>
    <Route exact path='/' component={ Home } />
    <Route path='/candle' component={ CandleChartPage } />
</Layout>;
