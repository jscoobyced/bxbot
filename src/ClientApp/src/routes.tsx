import * as React from 'react';
import { Route } from 'react-router-dom';
import { CandleChartPageHoc } from './components/charts/CandleChartPageHoc';
import { Home } from './components/Home';
import { Layout } from './components/Layout';

export const routes = <Layout>
    <Route exact path='/' component={Home} />
    <Route path='/candle' component={CandleChartPageHoc} />
</Layout>;
