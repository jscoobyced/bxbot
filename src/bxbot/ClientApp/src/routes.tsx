import * as React from 'react';
import { Route } from 'react-router-dom';
import { CandleChartPage } from './components/charts/CandleChartPage';
import { Home } from './components/Home';
import { Layout } from './components/Layout';

export const routes = <Layout>
    <Route exact path='/' component={ Home } />
    <Route path='/candle' component={ CandleChartPage } />
</Layout>;
