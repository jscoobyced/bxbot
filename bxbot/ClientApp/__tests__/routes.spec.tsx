import * as RoutesModule from '../routes';
import jest from 'jest';
import * as ShallowRenderer from 'react-test-renderer/shallow';

describe('routes', () => {
    const routes = RoutesModule.routes;
    it('should match snapshot', () => {
        const renderer = ShallowRenderer.createRenderer()
        expect(renderer.render(routes)).toMatchSnapshot();
    })
});