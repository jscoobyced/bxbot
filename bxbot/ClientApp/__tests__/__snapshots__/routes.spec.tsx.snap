// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes should match snapshot 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-sm-1"
    >
      <NavMenu />
    </div>
    <div
      className="col-sm-11"
    >
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/candle"
      />
    </div>
  </div>
</div>
`;
